bundle/manifests/windows-machine-config-operator_v1_serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
we cannot add service account file in bundle manifest, it causes issues with duplicate SA bundle validation in upgrade case.